srHook/global.ini

```diff
@@ -4,5 +4,12 @@ provider = SAPHanaSR
 path = /usr/share/SAPHanaSR
 execution_order = 1
 
+[ha_dr_provider_SAPHanaSrCostOptMemConfig]
```

Comment:
Should we separate the examples? So a global.ini per hook?

Reply:
That sounds really good. Especially when taking into account our idea of consolidating the coding and shipping of ScaleOut and ScaleUp.